package.json

```diff
@@ -33,21 +33,24 @@
     "start": "npm run start:storybook"
   },
   "devDependencies": {
+    "@babel/cli": "^7.4.4",
+    "@babel/core": "^7.4.5",
+    "@babel/plugin-proposal-object-rest-spread": "^7.4.4",
+    "@babel/preset-env": "^7.4.5",
+    "@babel/preset-react": "^7.0.0",
     "@storybook/addon-a11y": "^3.4.2",
+    "@storybook/addon-actions": "3.4.12",
     "@storybook/addon-centered": "^3.4.2",
     "@storybook/addon-info": "^3.4.2",
     "@storybook/addon-jest": "^3.4.2",
     "@storybook/addon-knobs": "^3.4.2",
     "@storybook/addon-options": "^3.4.2",
-    "@storybook/addon-actions": "3.4.12",
     "@storybook/react": "^3.4.2",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
+    "babel-core": "7.0.0-bridge.0",
```

Comment:
no version more "stable" ? is it the last one ?

Reply:
this is the last one and also some dependency force me to install this because:  https://github.com/babel/babel-bridge